### ISSUE TITLE
Add a simple GUI

### ISSUE THREAD

**Issue by mjordan** [mjordan/islandora_scg#18]
A simple Drupal form that would allow users without access to drush would be a big improvement.

**Comment by mjordan** [comment]
I've got a simple GUI:

![islandora_scg_gui](https://cloud.githubusercontent.com/assets/403918/26381432/9d1c542c-3fd9-11e7-93be-5024a9db836b.png)

but using it instead of Drush will require some refactoring of the content-type classes.

**Comment by mjordan** [comment]
Merged into 7.x as of e02aa6a8e02b81416897ced46624e143378b593c. I'm not happy with the feedback the GUI gives the user -  the Ajax spinner just disappears. I haven't been able to find a way to issue a nice message to the user if we have an Ajax spinner. Any suggestions (and a PR if possible) would be appreciated.

**Comment by mjordan** [comment]
Also note to self: test once #16 is merged.
